### PR TITLE
LL-5305 - add currencyName in Send and Receive flow

### PR DESCRIPTION
**Pull request by valpinkman**
As for addAccounts, adds currencyName into tracking event in Send and Receive flow

### Type

Improvement

### Context

LL-5305

### Parts of the app affected / Test plan

Analytics

### CODE REVIEW

File: src/renderer/modals/Receive/Body.js [hunk 1]
```diff
@@ -175,6 +178,20 @@ const Body = ({
...
+
+      const currencyName = currency
+        ? currency.type === "TokenCurrency"
+          ? currency.parentCurrency.name
+          : currency.name
+        : undefined;
+
+      setCurrencyName(currencyName);
+    }
+  }, [account]);
```
Comment (gre) [inline src/renderer/modals/Receive/Body.js:193]: this shouldn't be an effect to me.

**Review by gre — CHANGES_REQUESTED** [review]

**Review by Arnaud97234 — DISMISSED** [review]
Event present on both Receive and Send flows.
No event on staking operations.
✅

File: src/renderer/modals/Receive/Body.js (hunk 1 above)
Comment (valpinkman) [reply src/renderer/modals/Receive/Body.js]: fixed

**Review by juan-cortes — DISMISSED** [review]
lgtm

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
@valpinkman image

In Send / Receive flows currencyName should be the token name instead of the parent Currency name.

### How to reproduce
- Send / Receive ERC20

**Review by Arnaud97234 — APPROVED** [review]
Ok for Currency as well as tokens.
✅

**Review by JunichiSugiura — APPROVED** [review]

**Pull request merged**